docusaurus/docs/cms/admin-panel-customization/homepage.md

```diff
@@ -0,0 +1,761 @@
+---
+title: Homepage customization
+description: Learn about the Strapi admin panel Homepage and how to customize it with widgets.
+toc_max_heading_level: 6
+tags:
+- admin panel
+- homepage
+- widgets
+- features
+---
+
+# Homepage customization
+
+The <Icon name="house" /> Homepage is the landing page of the Strapi admin panel. By default, it provides an overview of your content with 2 default widgets:
+
+- _Last edited entries_: Displays recently modified content entries, including their content type, status, and when they were updated.
+- _Last published entries_: Shows recently published content entries, allowing you to quickly access and manage your published content.
+
+<ThemedImage
+  alt="Homepage with default widgets"
+  sources={{
+    light: '/img/assets/admin-homepage/admin-panel-homepage.png',
+    dark: '/img/assets/admin-homepage/admin-panel-homepage_DARK.png',
+  }}
+/>
+
+These default widgets cannot currently be removed, but you can customize the Homepage by creating your own widgets.
+
+:::note
+If you recently created a Strapi project, the Homepage may also display a quick tour above widgets if you haven't skipped it yet.
+:::
+
+## Adding custom widgets
+
+To add a custom widget, you need to:
+
+- register it
+- create a widget component
```

Comment:
I'd argue the most common way for users to get custom widgets will be to install them from marketplace plugins

Reply:
Noted. I'll still go through details of creating a custom widget, but will add a note about this.